src/spatial/reaction_rates.jl

```diff
@@ -52,16 +52,21 @@ end
 
 update rates of all reactions in rxs at site
 """
-function update_rx_rates!(rx_rates::RxRates{F, M}, rxs, integrator,
-                          site) where {F, M <: AbstractMassActionJump}
-    u = integrator.u
+function update_rx_rates!(rx_rates::RxRates{F, M}, rxs, u::AbstractMatrix, integrator,
```

Comment:
If `F` and `M` aren't constrained why make it parametric like this? i.e. why not just take `rx_rates::RxRates`?

Reply:
There was no reason but oversight. Good catch.